agent/virus_total_agent.py

```diff
@@ -50,7 +50,8 @@ def process(self, message: msg.Message) -> None:
         file_content = file.get_file_content(message)
         if file_content is not None:
             if (
-                len(self.whitelist_types) != 0
+                self.whitelist_types is not None
```

Comment:
Would've just defaulted in the init: 
`self.whitelist_types = self.args.get("whitelist_types") or []`

Reply:
+1